src/main/java/com/yahoo/bullet/querying/FilterOperations.java

```diff
@@ -108,12 +108,13 @@
     private static TypedObject getTypedValue(BulletRecord record, Type type, Value value) {
         TypedObject result = null;
```

Comment:
No need to initialize to null

Reply:
Otherwise Intellij complains it "might not be initialized"

Reply:
Ok. It can't detect all code paths lead to its initialization for some reason before return